mail/management/commands/import.py

```diff
@@ -9,16 +8,8 @@ class Command(BaseCommand):
 	help = 'Import Account and Alias data from HTTP hosted JSON files'
 
 	def handle(self, *args, **options):
-		datas = []
 		for url in args:
-			try:
```

Comment:
This loop is important to first collect all data.

In case one of the URLs is unreachable these accounts will be in a bad state otherwise.

Reply:
Fixed in the latest commit, include changes for Django3 support